features/step_definitions/show_dishes_steps.rb

```diff
@@ -0,0 +1,21 @@
+Given("the following dishes exists") do |table|
+  table.hashes.each do |hash|
+    Dish.create!(hash)
+  end
+end
+
+Given("I visit the dish page") do
+  visit '/restaurants/show'
```

Comment:
Please use route helpers

Reply:
I've been struggling with this, as you can see it my commit-comments. Just fixed it!!! :D